src/platform/packages/shared/kbn-es-archiver/src/actions/load.ts

```diff
@@ -133,7 +133,7 @@ export async function loadAction({
   );
 
   // If we affected saved objects indices, we need to ensure they are migrated...
-  if (Object.keys(result).some((k) => k.startsWith(MAIN_SAVED_OBJECT_INDEX))) {
+  if (Object.keys(result).some((k) => ALL_SAVED_OBJECT_INDICES.includes(k))) {
```

Comment:
@gsoldevila could you share your thoughts on this change? Did we miss updating kbnArchiver last time?